tooltester/src/main/java/io/dockstore/tooltester/blacklist/BlackList.java

```diff
@@ -19,7 +19,7 @@
     public static boolean isNotBlacklisted(String entryId, String version) {
```

Comment:
Personal preference, I would call this method `isBlacklisted` and change the implementation and callers accordingly. It's just an extra (tiny) leap of logic for me to process the `Not`.